docs/docs/fault-domain-awareness.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Fault Domain Awareness
+---
+
+# Fault Domain Awareness
+
+## Overview
+
+A fault domain is a set of nodes that share similar failure and latency characteristics. Two nodes in the same fault domain are both affected by failure events within the domain. Placing nodes in more than one fault domain reduces the risk that a failure will affect both nodes.
+
+Marathon supports fault domain awareness, so that you can mitigate the risk of fault domain failure for high availability and configure your apps and pods to take advantage of region awareness to allow for increased capacity when needed.
+
+Marathon currently offers hierarchical 2-level fault domain: zone and region.
+	
+## Zone fault domains
+Zone fault domains offer a moderate degree of fault isolation because they share the same region. However, network latency between zones in the same region is moderately low (typically < 10ms).
+	
+For on-premise deployments, a zone would be a physical data center rack. 
+
+For public cloud deployments, a zone would be the "availability zone” concept provided by most cloud providers.
+	
+Spread your apps or pods across zones if you are latency-sensitive or if you need a moderate degree of high availability: you can place your apps in one region, but balance across zones.
+
+## Region fault domains
+
+Region fault domains offer the most fault isolation, though inter-region network latency is high (typically 50-100ms ). 
+	 
+For on-premise deployments, a region might be a data center.
+	 
+For public cloud deployments, most cloud providers expose a “region” concept.
+	 
+You can deploy your apps or pods in a specific region region based on the available capacity.
```

Comment:
I might suggest that we consistently just call them "instances" instead of apps or pods (it _is_, after all, actually the instances we balance)

Reply:
that makes a lot of sense.